.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+image: leanprovercommunity/mathlib:gitpod
+
+vscode:
+  extensions:
+    - jroesch.lean
+
+tasks:
+  - command: . /home/gitpod/.profile
```

Comment:
Shouldn't this have a `leanpkg configure` and `leanpkg get-mathlib-cache`? (Or maybe just one of them)

Reply:
I don't know! I have no idea how gitpod works.

Reply:
Ah, yeah. The call to the script you deleted would have done this in `lean-liquid`. https://github.com/leanprover-community/lean-liquid/blob/master/scripts/get-cache.sh

You need at least `leanproject get-mathlib-cache`. I don't know what directory this runs in, you might need the `cd` too.